abs.yaml

```diff
@@ -0,0 +1,3 @@
+upload_channels:
+  - sfe1ed40
+  - https://staging.continuum.io/prefect/fs/colorlover-feedstock/pr2/1f8ae7c
```

Comment:
staging should not be in channels ?

Reply:
remember to remove this line before merging

Reply:
It's here https://github.com/santosjorge/cufflinks/blob/b973f3a0f689816139de6c68855ab65ffec15567/cufflinks/plotlytools.py#L1432-L1437

Reply:
Done